Documentation/cli-installation-scenarios.md

```diff
@@ -75,45 +73,40 @@ Removing the bits from the machine **must** follow the order outlined above in i
+Since this is the case, there is no special "upgrade". When the user needs a new version, the user just installs the new version using any of the installers specified in this document. The installer will just drop a new version at the predefined location. 
 
-This, however, does have one specific constraint: **newever versions must be installed in the same location the previous version was in**. This constraint is due to the fact that the "muxer" uses convention to figure out how to find the actual driver that the user reuqested. 
+This, however, does have one specific constraint: **newer versions must be installed in the same location the previous version was in**. This constraint is due to the fact that the "muxer" uses convention to figure out how to find the actual driver that the user reuqested. 
 
 ## Layout on disk
 ```
-~/dotnet
-	-dotnet (%PATH%)
-	-SDK
-       - 1.0.0
-           - 
-       - 2.0.0
-	-Shared
-		-[redist framework]
-			-1.0.0
-				-coreclr.dll
-				-mscorlib.ni.dll <Regular NI image>
-				-system.*.dll  <R2R NI image>
-				-dotnet-hostimpl.dll
-				-dotnet
-				-netcoreapp.deps.json 
-			-1.5.0
-				-coreclr.dll
-				-mscorlib.ni.dll <Regular NI image>-
-				-system.*.dll  <R2R NI image>
-				-dotnet-hostimpl.dll
-				-dotnet
-				-netcoreapp.deps.json 
+<INSTALL_DIR>/ (%PATH%)
+	dotnet ("muxer", has platform dependant file extension)
+    hostfxr (implementation detail for "muxer", platform dependant file extension)
+	sdk/
+        <sdk-version-0>/ (i.e. "1.0.0-rc2-002543")
+            ... (binaries like: dotnet.dll, csc.dll)
+        <sdk-version-1>/
+            ... (binaries)
+        ...
+	shared/ ("redist" or "shared framework")
+        <target-framework-name>/ (currently only "Microsoft.NETCore.App")
+			<redist-version-0>/ (i.e. "1.0.0-rc2-3002543")
+                ... (binaries like: coreclr.dll, mscorlib.ni.dll, System.*.dll, dotnet-hostimpl.dll, dotnet, netcoreapp.deps.json)
+			<redist-version-1>/
+				... (binaries)
 ```	
 
 ## Native dependencies
 .NET Core CLI is built on top of CoreFX and CoreCLR and as such its' dependencies set is defined by the platform that those two combine. Whether or not those dependencies will be installed depends on the installer being used. The table below lists out the installers and whether or not they bring in dependencies. 
 
-| Installer  	| Dependencies Y/N   	|
-|------------	|--------------------	|
-| EXE        	| Y (chains them in) 	|
-| PKG        	| N                  	|
-| apt-get    	| Y                  	|
-| rpm        	| Y                  	| 
+| Installer  	| Dependencies Y/N                   |
+|------------	|----------------------------------- |
+| EXE        	| Y (chains them in) 	             |
+| PKG        	| N (need to be manually installed*) |
+| apt-get    	| Y                                  |
+| rpm        	| Y                                  |
+
+`*` PKG has no way of installing native dependencies
```

Comment:
Not really true. It cannot specify and automatically install dependencies. It can have native dependencies and install them if we write the needed support.